corehq/motech/forms.py

```diff
@@ -46,10 +46,15 @@ class Meta:
 
     def __init__(self, *args, domain, **kwargs):
         self.domain = domain  # Passed by ``FormSet.form_kwargs``
+        instance = kwargs.get('instance', None)
```

Comment:
what happens if there is not `instance`

Reply:
This is just setting the value that gets sent to the browser when the form is opened. If the form is not for an existing instance, the form will be blank and the password field will not be populated.